src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -996,6 +996,17 @@ One of the most important notions in workflows is the dependencies between tasks
 print('Hello world')
 ```
 
+===== Variables
+
+To specify https://doc.activeeon.com/latest/user/ProActiveUserGuide.html#_task_variables[task variables],
+you should provide the `variables` parameter. Its value should be a list of tuples `(key,value)` that corresponds to
```

Comment:
you have to provide (not should)

Reply:
I prefer to keep "should" to show that this parameter is optional.